### ISSUE TITLE
ANR from Google Play console

### ISSUE THREAD

**Issue by jesmrec** [owncloud/android#2824]
Latest checks to Google Play console show that the following ANR is growing up:

```
"main" tid=1 Native
"main" prio=5 tid=1 Native
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x712a0b38 self=0x76f2c10800
  | sysTid=8612 nice=-10 cgrp=default sched=0/0 handle=0x77794fbed0
  | state=S schedstat=( 1411256161 423322433 2929 ) utm=95 stm=46 core=4 HZ=100
  | stack=0x7fc60fb000-0x7fc60fd000 stackSize=8192KB
  | held mutexes=
 
  #00  pc 00000000000c1228  /apex/com.android.runtime/lib64/bionic/libc.so (__epoll_pwait+8)
  #01  pc 00000000000180bc  /system/lib64/libutils.so (android::Looper::pollInner(int)+144) 
  #02  pc 0000000000017f8c  /system/lib64/libutils.so (android::Looper::pollOnce(int, int*, int*, void**)+56) 
  #03  pc 000000000013d05c  /system/lib64/libandroid_runtime.so (android::android_os_MessageQueue_nativePollOnce(_JNIEnv*, _jobject*, long, int)+44) 
  at android.os.MessageQueue.nativePollOnce (Native method) 
  at android.os.MessageQueue.next (MessageQueue.java:336) 
  at android.os.Looper.loop (Looper.java:174) 
  at android.app.ActivityThread.main (ActivityThread.java:7682) 
  at java.lang.reflect.Method.invoke (Native method) 
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:516) 
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:950)
```

If's happening in 2.14.2 version of the app, in devices with Android 9 an 10

I think this one deserves a look

**Comment by abelgardep** [comment]
I was able to reproduce an anr creating some new folders using DocumentsProvider. It may be fixed in #2846.

**Comment by jesmrec** [comment]
Thanks @abelgardep! will be tested in the scope of such PR. After the official release, we must have an eye in the number of crash reports to check if that one keeps happening or not.

Also, this improvement: https://github.com/owncloud/android/issues/2798 , should be a high priority in the following milestone.
